runtime/src/main/resources/application.properties

```diff
@@ -68,4 +68,5 @@ multiple.providers.black.properties=OS_AUTH_URL
 huaweicloud.international.price.calculator.url=https://portal-intl.huaweicloud.com/api/cbc/global/rest/BSS/billing/ratingservice/v2/inquiry/resource
 huaweicloud.european.price.calculator.url=https://portal.eu.huaweicloud.com/api/cbc/global/rest/BSS/billing/ratingservice/v2/inquiry/resource
 huaweicloud.chinese.price.calculator.url=https://portal.huaweicloud.com/api/cbc/global/rest/BSS/billing/ratingservice/v2/inquiry/resource
-enable.agent.api.only=false
\ No newline at end of file
+enable.agent.api.only=false
+javax.persistence.lock.timeout=2000
```

Comment:
we need a better name here which is specific just for the agent wait time out.
pending.config.change.query.wait.time